resources/translations/en.edn

```diff
@@ -266,6 +266,7 @@
                 :column-n "Column %1"
                 :column-key "ID code"
                 :column-label "Label"
+                :field-id "Field ID code"
```

Comment:
Maybe shorten to just field id?

Reply:
I was trying to mirror the "ID code" we use for options & table columns

Reply:
Well the Swedish translation does not follow the same "code"? And it seems overly complicated (perhaps it was not really thought about for the options either).

Reply:
Yeah for swedish I was mirroring the `:option-key "Id"` translation that was already there. I'm fine with dropping "code" from all the english localizations.